MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -33,13 +33,13 @@
             </div>
         </div>
         <div class="col-sm-5">
-            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Mohini">
+            <img src="\images\Mohini.png" class="card-img-top h-75" alt="Mohini" />
```

Comment:
Added end tag in the img src tag because empty elements must always be closed
https://www.w3schools.com/html/html_xhtml.asp